fido_app/utils.py

```diff
@@ -35,3 +35,28 @@ def get_first_result(dbcursor):
     first_result = results[0].fetchone()
 
     return first_result
+
+
+def validate_email(email):
+    '''
+    Returns True if email is a properly formatted email address;
+    returns False otherwise.
+    '''    
+    return isinstance(email, str) and re.search('^([a-z0-9]+[\._]?)*[a-z0-9]+@\w+\.\w+$', email) is not None
+
+
+def validate_display_name(name):
+    '''
+    Returns True if name is a properly formatted display name,
+    which at the moment means name contains at least one word character;
+    returns False otherwise.
+    '''
+    return isinstance(name, str) and re.search('\w') is not None
```

Comment:
Two things: looks like you forgot to search through the name in the second part of the condition and you should make sure to use [raw strings](https://docs.python.org/3/library/re.html#raw-string-notation) with regular expressions 
that have backslashes.
```suggestion
return isinstance(name, str) and re.search(r'\w', name) is not None
```

Reply:
I really am getting sloppy...fixed this in d0b7d729ef14de13ced61a4b2a46370a723abc90.